### DISCUSSION TITLE
Revisit Date/Time fields

### DISCUSSION THREAD

**Discussion by JoryHogeveen** [pods-framework/pods#6721] [Ideas]
- [ ] Combine the three fields into one
- [ ] I18n JS (should be here, not in #3641)
- [ ] Validate existing options for JS and HTML5
- [ ] Enhance options for JS and HTML5
- [ ] Check/extend output options (surgestions please!).
    - [ ] #3927 
    - [ ] Maybe allow a custom input for format?
- [ ] Feature: HTML5 Date options for date and datetime fields
(From Slack)
@yourpcmagician14
Ok so Safari and IE apparently don't honor minDate. I changed the code slightly  to remove the minYear and maxYear options and did $args[ 'yearRange' ] = '1900:2016';

Also see https://github.com/pods-framework/pods-code-library/blob/master/example/hooks/filters/pods_form_ui_field_date_args/examples/date-range.php

**Comment by JoryHogeveen** [comment]
Maybe also just combine `datetime.php`, `date.php` and `time.php` into one field type where the user has the option to enable/disable time and date functionality.

**Comment by JoryHogeveen** [comment]
Also change HTML input type `datetime` to `datetime-local` for browser support. `datetime` is removed.
See "datetime" input at: http://www.w3schools.com/html/html_form_input_types.asp

> The input type datetime is removed from the HTML standard. Use datetime-local instead.

**Comment by JoryHogeveen** [comment]
Also add `week` & `month` in there

**Comment by jimtrue** [comment]
Fuzzy date input logic? Next week, 3 weeks from today

**Comment by jimtrue** [comment]
Some people also use just 'parts' of the field (ie Year). To me a full datetime string is always more useful. It's just what it 'stores' that matters.

**Comment by quasel** [comment]
putting that on the DFV list for 2.8 to comb through all the field issues ...

**Comment by sc0ttkclark** [comment]
Date/time/datetime fields have been rewritten for DFV in React, we should review what's left to be done here at some point soon.
